app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='expense'>
+  <header class='header'>
+    <h1><%= @category.name %></h1>
+    <span class='total'><%= number_to_currency(@total) %></span>
+  </header>
+
+  <ul class='expense_list'>
+    <% @expenses.each do |expense| %>
+      <li class='expense_item'>
+        <div class='expense_item-grid'>
+          <div class='right'>
+            <div>
+              <span class='expense_name'><%= expense.name %></span>
+              <span class='expense_amount'><%= number_to_currency(expense.amount) %></span>
+            </div>
+
+            <span class='expense_date'><%= expense.created_at.strftime('%b %d, %Y') %></span>
+          </div>
+        </div>
+      </li>
+    <% end %>
```

Comment:
- [ ] Great job working on the expense so far but I recommend to change the color of the buttons as well here as mentioed in the requirements so I highly recommend to change it 💪🏻🔥


## the nav is blue and the button bg is green , here's the main colors of the project

![image](https://github.com/Timbar09/Budget-App/assets/58553711/ac1d1402-566b-4b04-a25e-e0cdcb24640a)


- Your
![image](https://github.com/Timbar09/Budget-App/assets/58553711/69bbdd03-5ea4-4c2a-80b0-1ad74a46a7ca)

- Given
![image](https://github.com/Timbar09/Budget-App/assets/58553711/d41d4890-390c-417a-95a9-a5f8cec946da)